docs/UserGuide.md

```diff
@@ -160,6 +122,89 @@ Expected output (fail):
 ```agsl
 Oops! Invalid search expression, please check again.
 ```
+### Viewing a fosterer's detail : `view`
+
+Views a fosterer's details by shifting the foster family list view page to a fosterer's profile view page. 
+
+Format: `view INDEX` 
+
+Parameters:
+* `INDEX`: Index of a fosterer to view their details shown in the list obtained by the find/list command
+
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+Some commands are not available while on profile view page. 
+The list of available commands are: 
+<ul>
+  <li> <code>help</code> </li>
+  <li> <code>edit</code> </li>
+  <li> <code>exit</code> command to <b>exit the profile view page</b> </li>
+</ul>
+</div> 
+<br>
+
+Example: 
+* `list` followed by `view 2` to view the profile of the 2nd fosterer in the address book. 
+
+Expected output (success):
+```agsl
+Viewing the profile of a fosterer...
+```
+
+Expected output (fail):
+```agsl
+Oops! Invalid fosterer index provided, please check again.
+```
+
+### Editing a fosterer's detail : `edit`
+
+Edits or views the details of a fosterer stored in the address book. 
+
+Format 1: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [animal/ANIMAL_NAME] [t/AVAILABILITY] [t/HOUSING_TYPE] [t/TYPE_OF_ANIMAL_FOSTERED/WANTED…]`
```

Comment:
Yup, they are no longer tags! I will update the add command UG part after we merge everything, then I'll inform brandon about the changes if theres a need for clarification. Otherwise, the last message on tele about the add command is correct.